packages/FormElement/stories/FormElement.stories.mdx

```diff
@@ -0,0 +1,265 @@
+
+## More Examples
+
+### Index
+
+- <a href="#accessibility">Accessibility</a>
+- <a href="#alternate-layouts">Alternate Layout</a>
+
+  - <a href="#checkboxes">Checkboxes</a>
+  - <a href="#radios">Radios</a>
+  - <a href="#date-picker">Date Picker</a>
+  - <a href="#listbox">ListBox</a>
+  - <a href="#buttongroup">ButtonGroup</a>
+  - <a href="#nested-formelement">Nested FormElements</a>
+
+### Accessibility
+
+The Formelement has been designed with accessibiklity in mind. By using `<FormElement.Instructions>` users can add instructions for the end user. Users can also make use of the `<FormElement.Description>` to add additional direction for consumers.
+
+```jsx
+<FormElement label="Form Label">
+  <FormElement.Instructions>
+    <span>
+      Example text for extra panel for questionnaires. Example text for extra panel for questionnaires Example text for
+      extra panel for questionnaires. Example text for extra panel for questionnaires
+    </span>
+  </FormElement.Instructions>
+  <FormElement.Content>
+    {({ idForLabel, ariaDescribedBy }) => (
+      <Input
+        id={idForLabel}
+        onChange={handleChange}
+        value={value}
+        placeholder="Form placeholder"
+        aria-describedby={ariaDescribedBy}
+        hasError={Boolean(errorText.length)}
+      />
+    )}
+  </FormElement.Content>
+  <FormElement.Description>
+    <span>This is description text</span>
+  </FormElement.Description>
+  <FormElement.Error>{errorText}</FormElement.Error>
+  <FormElement.Help>
+    Give me some help. <a href="wegalvanize.com">Learn more</a>.
+  </FormElement.Help>
+</FormElement>
+```
+
+##### Result
+
+<FormElementStory>
+  <AccessibilityExample />
+</FormElementStory>
+
+### Alternate Layouts
+
+The FormElement can be used in a variety of formats. These can include FormElements with checkboxes, radios, listbox's etc.
+
+#### Checkboxes
+
+```jsx
+<FormElement hasFieldSet label="Form Label">
+  <FormElement.Content>
+    {({ ariaDescribedBy: ariaDescribedByOuter }) => getCheckboxOptions(ariaDescribedByOuter)}
+  </FormElement.Content>
+  <FormElement.Description>
+    <span>Description of this field.</span>
+  </FormElement.Description>
+  <FormElement.Error>{errorText}</FormElement.Error>
+  <FormElement.Help>
+    Give me some help. <a href="wegalvanize.com">Learn more</a>.
+  </FormElement.Help>
+</FormElement>
+```
+
+##### Result
+
+<CheckboxExample />
+
+#### Radios
+
+```jsx
+<FormElement hasFieldSet label="Form Label">
+  <FormElement.Content>
+    {({ ariaDescribedBy: ariaDescribedByOuter }) => getRadioOptions(ariaDescribedByOuter)}
+  </FormElement.Content>
+  <FormElement.Description>
+    <span>Description of this field.</span>
+  </FormElement.Description>
+  <FormElement.Error>{errorText}</FormElement.Error>
+  <FormElement.Help>
+    Give me some help. <a href="wegalvanize.com">Learn more</a>.
+  </FormElement.Help>
+</FormElement>
+```
+
+##### Result
+
+<RadioExample />
+
+#### Date Picker
```

Comment:
Similarly, I think we can have an example for more complex input components (DatePicker, ListBox, ButtonGroup, etc).  Let's explain that, show one code snippet (you can pick which one you think is the most helpful) and then maybe show all the results... or have some links to the other example stories (or maybe just the example story source code on GitHub).